src/js/analyses/components/Create/SubtractionSelector.js

```diff
@@ -28,14 +28,26 @@ export const SubtractionSelector = ({ subtractions, selected, onChange }) => {
                 <CreateAnalysisSelectorList
                     items={unselectedSubtractions}
                     render={({ id, name }) => (
-                        <SubtractionSelectorItem key={id} id={id} name={name} onClick={handleClick} />
+                        <SubtractionSelectorItem
+                            key={id}
+                            id={id}
+                            name={name}
+                            onClick={handleClick}
+                            isDefault={includes(defaultSubtractions, id) ? true : false}
```

Comment:
I'd like to extract this to a new select in `src/js/analyses/selectors` called `getAnalysisSubtractions()`.

Use reselect to write a selector that returns the subtractions with an added `default` field on each object. This will avoid calling `includes` (expensive) for every subtraction on every render.

There a more efficient solution than `includes`, but putting this behind reselect will be good enough.